spec/index.bs

```diff
@@ -27,6 +27,14 @@ Introduction {#intro}
 
 This specification rocks.
 
+Convention {#convention}
+========================
+
+## Coordinate Systems ## {#coordinate-systems}
+WebGPU's coordinate systems match DirectX and Metal's coordinate systems in graphics pipeline.
+  - Y-axis is up in normalized device coordinate (NDC): point(-1.0, -1.0) in NDC is located at the bottom-left corner.
+    In addition, x and y in NDC must be between -1.0 and 1.0 inclusive, while z in NDC must be between 0.0 and 1.0 inclusive.
```

Comment:
What "x", "y", and "z" are being constrained here, and what happens if they don't fulfill the constraint?

Reply:
They're probably clipped, but we can specify this later.

Reply:
I think this whole line is confusing here. When I read "Xxx must be" I'm thinking about the validation rules, which is not the case here at all. When I see "inclusive", I start wondering about what difference it makes. We should either write this down correctly or omit the line completely (in favor of a different section that explains clipping).